invokeai/app/invocations/latent.py

```diff
@@ -385,8 +423,16 @@ class LatentsToLatentsInvocation(TextToLatentsInvocation):
     type: Literal["l2l"] = "l2l"
 
     # Inputs
+    noise: Optional[LatentsField] = Field(description="The noise to use (test override for future optional)")
+
+    denoising_start: float = Field(default=0.0, ge=0, le=1, description="")
+    #denoising_end: float = Field(default=1.0, ge=0, le=1, description="")
```

Comment:
Now that `strength` is a userland concept:
- We need a formulas for `strength` <-> `denoising_start` implemented in JS. I know they are very simple, just making a note.
- We need to apply those formulas not only when creating graphs, but also when recalling metadata
- Do we change metadata to only have `denoising_start`? I think so